tasks/main.yml

```diff
@@ -17,3 +17,13 @@
     src: etc/sudoers.d/template
     dest: "/etc/sudoers.d/{{ item }}"
   loop: "{{ administrators_names }}"
+  
+- name: remove user
+  user:
+    name: "{{ item.name }}"
+    shell: "{{ administrators_default_shell }}"
+    state: absent
```

Comment:
We also want to add the users only if they are `active`

Reply:
done